miden-tx/src/lib.rs

```diff
@@ -40,6 +40,9 @@ pub use error::{
     TransactionProverError, TransactionVerifierError, KERNEL_ERRORS,
 };
 
+mod signatures;
+pub use signatures::get_falcon_signature;
```

Comment:
I would probably make the module public - so, it would look something like:
```Rust
pub mod signatures;
```
And then there is no need to re-export a specific function.

Another option could be to have an `auth` module and move things like `TransactionAuthenticator`, `BasicTransactionAuthenticator`, and `get_falcon_signature` into this module and then export this module publicly. This way, all authentication-related components would be in a single top-level module - but maybe I'm overthinking it.

Reply:
Done. Left the `signatures` module as a submodule of `auth`.